bedrock/privacy/urls.py

```diff
@@ -29,6 +29,7 @@
     path("firefox-relay/", views.firefox_relay_notices, name="privacy.notices.firefox-relay"),
     path("mozilla-vpn/", views.mozilla_vpn, name="privacy.notices.mozilla-vpn"),
     path("mdn-plus/", views.mdn_plus, name="privacy.notices.mdn-plus"),
+    path("ad-targeting-guidelines/", views.ad_targeting_guidelines, name="privacy.notices.ad-targeting-guidelines"),
```

Comment:
**question (potentially blocking)**: do we need to add a link for this page in the `base-protocol` side nav?

Reply:
That's not been requested at this time but I'll ask the question, thanks for calling out 👍